RPS.rb

```diff
@@ -0,0 +1,29 @@
+require "pry"
+
+class RockPaperScissors
+
+def winner(p1,p2)
+        @p1 = p1
+        @p2 = p2
+
+        # p = (@p1[1,1]+@p2[1,1]).sort
+        p = (@p1[1]+@p2[1]).sort
```

Comment:
Use more semantic variable names. Would make this line (and the method) a lot more readable!